yaml_files/simphony_metadata.yml

```diff
@@ -444,38 +444,34 @@ CUDS_KEYS:
   ######################################################
   # Conditions on Boundaries
   ######################################################
-  DIRICHLET:
-    definition: Dirichlet boundary condition, specify the value the solutions takes on the boundary of the domain
+  VALUE_CONDITION:
+    definition: Boundary condition encountered in the solution of partial differential equations
     parent: CUBA.CONDITION
     models: [CUBA.CONTINUUM]
     CUBA.VARIABLE:
-      shape: (:)
+      shape: [1]
       default: []
-    CUBA.MATERIAL:
+    CUBA.VALUE:
       shape: (:)
+      default: 0.0
+    CUBA.MATERIAL:
+      shape: [1]
       default: []
 
+
+
+  DIRICHLET:
+    definition: Dirichlet boundary condition to specify the value the solutions takes on the boundary of the domain.
+    parent: CUBA.VALUE_CONDITION
+
   NEUMANN:
-    definition: Neumann boundary condition, it specifies the values that the derivative of a solution is to take on the boundary of the domain.
-    parent: CUBA.CONDITION
-    models: [CUBA.CONTINUUM]
-    CUBA.VARIABLE:
-      shape: (:)
-      default: []
-    CUBA.MATERIAL:
-      shape: (:)
-      default: []
+    definition: Neumann boundary condition, it specifies the values that the derivative of a solution with respect to the variable is to take on the boundary of the domain.
+    parent: CUBA.VALUE_CONDITION
 
   EMPTY:
-    parent: CUBA.CONDITION
     definition: Empty boundary condition
-    models: [CUBA.CONTINUUM]
-    CUBA.VARIABLE:
-      shape: (:)
-      default: []
-    CUBA.MATERIAL:
-      shape: (:)
-      default: []
+    parent: CUBA.VALUE_CONDITION
+    value: CUBA.NONE
```

Comment:
it merely means that there is no default. CUBA.NONE is defined in cuba.yaml. I assume that so far an empty default is equivalent to NONE?